data/json/items/gunmod/muzzle.json

```diff
@@ -21,7 +21,7 @@
     "id": "crafted_suppressor",
     "type": "GUNMOD",
     "name": "homemade suppressor",
-    "description": "A suppressor reduces the amount of noise and muzzle flash generated by the report of a firearm.  Gunfire is extremely loud and can damage your hearing without protection; a suppressor will bring the loudness of a report down to generally safe levels.  A suppressor will slightly reduce recoil and muzzle velocity.",
+    "description": "A suppressor reduces the amount of noise and muzzle flash generated by the report of a firearm.  Gunfire is extremely loud and can damage your hearing without protection; a suppressor will bring the loudness of a report down to generally safe levels.  A suppressor will slightly reduce recoil and muzzle velocity. This suppressor is large and blocks your view. You will not be able to aim down the base sights of the gun with this suppressor attached. Install new sights to be able to aim with this suppressor attached.",
```

Comment:
The original description was a word-for-word copy of the regular suppressor. Consider adding some flavor to this to connote it being homemade.

Perhaps:

> A crude, homemade suppressor that reduces the amount of noise and muzzle flash generated by the report of a firearm.  Gunfire is extremely loud and can damage your hearing without protection; a suppressor will bring the loudness of a report down to generally safe levels, as well as slightly reducing recoil and muzzle velocity. This simple suppressor is large and, when attached, will prevent you from aiming down the base sights of a gun. Install new sights to be able to aim with this suppressor attached.

Adding a note of how this isn't quite as good as a proper suppressor would be nice as well, but it's already quite wordy.